tiledb/sm/query/readers/reader_base.h

```diff
@@ -290,7 +290,7 @@ class ReaderBase : public StrategyBase {
   void zero_out_buffer_sizes();
 
   /** Correctness checks for `subarray_`. */
```

Comment:
Rename remote_query to check_ranges_oob as the legacy reader does it only for dense.